account_payment_term_extension/views/account_payment_term.xml

```diff
@@ -1,5 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
+    <record id="view_payment_term_holiday_tree" model="ir.ui.view">
+        <field name="name">Tree view for account.payment.term.holiday</field>
+        <field name="model">account.payment.term.holiday</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="holiday"/>
```

Comment:
adding the payment term would help to see to whom is assigned

Reply:
Done